index.js

```diff
@@ -29,9 +29,9 @@ exports.handler = function(event, context, callback) {
     .set(headers)
     .end((err, res) => {
       if (err) {
-        console.log('error', err);
+        return callback(err);
       } else {
-        console.log('success', res);
```

Comment:
Avoid if/else, instead return early
https://blog.timoxley.com/post/47041269194/avoid-else-return-early
So instead of 
```js
if (err) {
  console.log("error", err);
  return callback(err);
} else {
  console.log("success", res);
  return callback(res);
}
```
Do
```js
if (err) {
  console.log("error", err);
  return callback(err);
}
console.log("success", res);
return callback(res);
```

Reply:
updated as illustrated